install/kubernetes/helm/istio/values.yaml

```diff
@@ -420,6 +420,9 @@ global:
   #   else:  default dns domain
   trustDomain: ""
 
+  # (optional) Specify the mesh UID.
```

Comment:
Comments please. What does it mean ? What will it do ? How to set it in MC ? This is the doc that the user will see when installing.